src/app/shared/navBar/navBarDirective.js

```diff
@@ -0,0 +1,13 @@
+import './navBarStyles.styl'
+
+angular.module('app')
+  .directive('navBar', function ($log) {
+    return {
+      template: require('./navBarView.html'),
+      restrict: 'A',
+      scope: {
+        flag: '='
+      }
+    }
+  }
+)
```

Comment:
Does this directive have a parameter on it called 'flag'? If not, let's take it out before committing it.